app/views/shared/_google_analytics_settings.html.erb

```diff
@@ -0,0 +1,92 @@
+<!-- Google Analytics Settings Section -->
+<div class="card mb-4 border-left-primary">
+  <div class="card-header bg-light">
+    <h5 class="mb-0 text-primary">
+      <i class="fas fa-chart-line mr-2"></i> Google Analytics 4 Configuration
+    </h5>
+  </div>
+  <div class="card-body">
+    <% if settings.key?(:analytics) %>
+      <div class="form-group mb-4">
+        <%= render 'shared/settings', f: f, key: :analytics, value: settings[:analytics] %>
+      </div>
+    <% end %>
+    
+    <div class="row" id="google-analytics-fields" style="display: none;">
+      <% if settings.key?(:google_analytics_id) %>
+        <div class="col-md-6">
+          <%= render 'shared/settings', f: f, key: :google_analytics_id, value: settings[:google_analytics_id] %>
+        </div>
+      <% end %>
+      <% if settings.key?(:google_analytics_property_id) %>
+        <div class="col-md-6">
+          <%= render 'shared/settings', f: f, key: :google_analytics_property_id, value: settings[:google_analytics_property_id] %>
+        </div>
+      <% end %>
+    </div>
+  </div>
+</div>
+
+<script>
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
live in the view is not where javascript goes =-p

Reply:
totally agree 🙃 I'll move it in my clean up iteration.